System/Posix/ARX/CLI/Options.hs

```diff
@@ -98,6 +101,8 @@ qPath                        =  tokCL QualifiedPath
 shared                      ::  ArgsParser Bool
 shared                       =  True <$ arg "--shared"
 
+tmpdir                       :: ArgsParser Path
```

Comment:
`␣::␣ArgsParser Path` -> `::␣␣ArgsParser Path` for consistency with surrounding code.